src/reanimated2/interpolation.js

```diff
@@ -1,30 +1,115 @@
 import { Extrapolate } from '../derived/interpolate';
 
+function getVal(config) {
+  'worklet';
+
+  const { type, coef, val, ll, rr, x } = config;
+
+  switch (type) {
+    case Extrapolate.IDENTITY:
+      return x;
+    case Extrapolate.CLAMP:
+      if (coef * val < coef * ll) {
+        return ll;
+      }
+      return rr;
+    case Extrapolate.EXTEND:
+    default:
+      return val;
+  }
+}
+
+function isExtrapolate(value) {
+  'worklet';
+
+  return (
+    value !== Extrapolate.EXTEND &&
+    value !== Extrapolate.CLAMP &&
+    value !== Extrapolate.IDENTITY
+  );
```

Comment:
Could you please invert the logic here?

```
value === Extrapolate.EXTEND ||
value === Extrapolate.CLAMP ||
value === Extrapolate.IDENTITY
```

is way easier to read.

Reply:
Done ✅